internal/authtoken/repository.go

```diff
@@ -289,16 +297,17 @@ func (r *Repository) ListAuthTokens(ctx context.Context, withOrgId string, opt .
 // DeleteAuthToken deletes the token with the provided id from the repository returning a count of the
 // number of records deleted.  All options are ignored.
 func (r *Repository) DeleteAuthToken(ctx context.Context, id string, opt ...Option) (int, error) {
+	const op = "authtoken.(Repository).DeleteAuthToken"
 	if id == "" {
-		return db.NoRowsAffected, fmt.Errorf("delete: auth token: missing public id: %w", errors.ErrInvalidParameter)
+		return db.NoRowsAffected, errors.New(errors.InvalidParameter, op, "missing public id")
 	}
 
 	at, err := r.LookupAuthToken(ctx, id)
 	if err != nil {
 		if errors.IsNotFoundError(err) {
 			return db.NoRowsAffected, nil
 		}
-		return db.NoRowsAffected, fmt.Errorf("delete: auth token: lookup %w", err)
+		return db.NoRowsAffected, errors.Wrap(err, op, errors.WithCode(errors.InvalidParameter))
```

Comment:
`LookupAuthToken` was refactored in this PR to return a valid domain error no need to wrap with code